src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -287,7 +293,57 @@ def validate
         false
       end
 
-    private
+      # Check if a system mount point is reused without formatting the
+      # partition during installation and warn the user if it is.
+      #
+      # @return [Boolean] true if okay, false if not
+      def validate_system_mount_points
+        return true unless Yast::Mode.installation
+        return true if to_be_formatted?
+        return true if !mounted?
+        return true unless ["/", "/usr", "/boot"].include?(mount_path)
+        warn_unformatted_system_mount_points
+      end
+
+      # Post a warning about reusing unformatted system mount points.
+      #
+      # @return [Boolean] true if the user wants to continue
+      def warn_unformatted_system_mount_points
+        log.info("warn_unformatted_system_mount_points")
+        # Translators: popup text
+        message = _(
+          "\n" \
+          "You chose to install onto an existing partition that will not be\n" \
+          "formatted. YaST cannot guarantee your installation will succeed,\n" \
+          "particularly in any of the following cases:\n"
+        ) +
+          # continued popup text
+          _(
+            "- if this is an existing ReiserFS partition\n" \
+            "- if this partition already contains a Linux distribution that will be\n" \
+            "overwritten\n" \
+            "- if this partition does not yet contain a file system\n"
+          ) +
+          # continued popup text
+          _(
+            "If in doubt, better go back and mark this partition for\n" \
+            "formatting, especially if it is assigned to one of the standard mount points\n" \
+            "like /, /boot, /opt or /var.\n"
+          ) +
+          # continued popup text
+          _(
+            "If you decide to format the partition, all data on it will be lost.\n" \
+            "\n" \
+            "Really keep the partition unformatted?\n"
```

Comment:
NP: The question is clear enough thanks to the previous explanation, but I would say something like "Really skip formatting the partition?".

I mean "keep the partition unformatted" sounds confusing when you actually are trying to reuse a formatted partition keeping the current format. There is nothing unformatted in such setup. Its simply not formatted by us.

Reply:
Oh, I saw now the trick below about keeping the old text. So never mind.